CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+CHANGES IN VERSION 1.18.0 (FROM 1.17.1)
+===================================
+
+Appium 1.18.0 is a minor release
+
+#### General
+* Add `score` attribute for ImageElement [appium-base-driver#396](https://github.com/appium/appium-base-driver/pull/396)
+* Add to proxy Chrome DevTools endpoint to downstream drivers [appium-base-driver#405](https://github.com/appium/appium-base-driver/pull/405)
+  * e.g. Clients can send `/session/:sessionId/goog/cdp/execute` command to Chromedriver
+* Fix socket leak by handling connections in a shared pool [appium-base-driver#416](https://github.com/appium/appium-base-driver/pull/416)
+* Fix to encode filenames with UTF-8 in extracting files from `.ipa` [appium-base-driver#419](https://github.com/appium/appium-base-driver/pull/419)
+  * This change fixes installing `.ipa` which bundles UTF-8 file names such as [this issue](https://github.com/appium/appium/issues/14100)
```

Comment:
The change fixes `.ipa` packages deployment that contain file names containing non-ASCII characters